src/test/java/uk/gov/pay/rule/PostgresTestDocker.java

```diff
@@ -55,4 +55,8 @@ public static String getDbPassword() {
     public static String getDbUsername() {
         return DB_USERNAME;
     }
+
+    public static void shutDown() {
+        POSTGRES_CONTAINER.stop();
```

Comment:
Might need to set this to null after stopping so that the next test can create a new container.

Reply:
👍